### ISSUE TITLE
Sorting columns with different units

### ISSUE THREAD

**Issue by zizzpudding** [grafana/explore-logs#563]
I know this is a known issue, but column sorting is not working correctly when there is a mix of units. 
In this case seconds and milliseconds. 

![image](https://github.com/grafana/explore-logs/assets/573085/1abb419b-70f5-40c1-a7cb-fdc5f3bdcdf3)

**Comment by gtk-grafana** [comment]
Yeah this is a known problem, unfortunately it doesn't have an easy fix unless we disable the ability to filter by these values, as these are the actual values in loki.

I tried fixing this by changing the display function of the duration field, but the table doesn't currently support rendering/sorting a display value that is different then the actual value.

Unless there's something we can do on the loki side to have it always return a numeric value in the same format (i.e. duration_ms) @cyriltovena?
